UIKitDZ/UIKitDZ/ViewController.swift

```diff
@@ -8,8 +8,163 @@
 import UIKit
 /// ViewController
 class ViewController: UIViewController {
```

Comment:
```suggestion
final class ViewController: UIViewController {
```
ViewController - название